### PR TITLE
Updating docs for BaseGame camera to clarify how it works

### PR DESCRIPTION
**Pull request by kurtome**
Relates to https://github.com/flame-engine/flame/issues/151

### CODE REVIEW

**Comment by erickzanardo** [comment]
Make your PR point to develop branch please.

File: doc/game.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ To start, just add your game widget directly to your runApp, like so:
...
 
-The `BaseGame` implements a `Component` based `Game` for you; basically it has a list of `Component`s and repasses the `update` and `render` calls appropriately. You can still extend those methods to add custom behavior, and you will get a few other features for free, like the repassing of `resize` methods (every time the screen is resized the information will be passed to the resize methods of all your components) and also a basic camera feature (that will translate all your non-HUD components in order to center in the camera you specified).
+The `BaseGame` implements a `Component` based `Game` for you; basically it has a list of `Component`s and repasses the `update` and `render` calls appropriately. You can still extend those methods to add custom behavior, and you will get a few other features for free, like the repassing of `resize` methods (every time the screen is resized the information will be passed to the resize methods of all your components) and also a basic camera feature. The `BaseGame.camera` controls which point in
+coordinate space should be the top-left of the screen (defaults to [0,0] like a regular `Canvas`).
```
Comment (luanpotter) [inline doc/game.md]: remove this extra line break :)

**Comment by kurtome** [comment]
@erickzanardo Gotcha, I think I fixed it. I made a new branch of develop, rebased the feature branch on it, the pushed it to this PR's branch.

**Comment by kurtome** [comment]
Thanks for the quick responses

**Pull request merged**